app/views/devise/sessions/new.html.erb

```diff
@@ -14,7 +14,7 @@
           <%#= link_to 'Sign up', new_registration_path(resource_name), class: 'right' %>
         <% end -%>
         <%= f.email_field :email,
-                          label: t('mongoid.attributes.user.email'),
```

Comment:
No objection here - but is this nonsequitor to the rest of the PR?

Reply:
eeeeeeeeeeh maybe. Sorry, might have gotten turned around on this one on including it, though it is part of the changeset - I'll keep it and be diligent about keeping stuff scoped to models going fwd.